myapp/spec/system/tasks_spec.rb

```diff
@@ -0,0 +1,166 @@
+require 'rails_helper'
+
+RSpec.describe Task, type: :system do
+  describe '#index' do
+
+    context "when task doesn't exists" do
+      before { visit root_path }
+
+      it "doens't show any task" do
+        expect(page).to have_no_button "Delete"
+      end
+    end
+
+    context 'when one task exists' do
+      let!(:task_1) { create(:task) }
+      before { visit root_path }
+
+      it 'shows one task' do
+        expect(page).to have_content task_1.title
+        expect(page).to have_content task_1.description
+        expect(page).to have_link task_1.title, href: task_path(task_1)
+        expect(page).to have_button "Delete"
+      end
+    end
+
+    context 'when multiple tasks exist' do
+      let!(:task_1) { create(:task) }
+      let!(:task_2) { create(:task, title: 'title 2', description: 'description 2') }
+      let!(:task_3) { create(:task, title: 'title 3', description: 'description 3') }
```

Comment:
This will reduce the code.
```suggestion
      let!(:tasks) { create_list(:task, 3) }
```

Reply:
Thank you for the suggestion. This is useful.